lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -54,13 +56,16 @@ class RenderForm extends React.Component {
     if (!course) {
       course = $course.val();
     }
-
+    if (!course) {
+      $('#course').closest('.form-group').addClass('has-error');
+      errors.push(gettext('Select a course for your support request.'));
```

Comment:
This message could be better if it states something like "Select a course or select  "Not specific to course" option".

Reply:
If we want to keep this, we should move this code in `validateData` method, where other fields are validated.

Reply:
message still needs to be updated, also it will be great if you can move this check into `validatedData`

Reply:
I'll update the message and I think this validation can live here because it needs less code here.